source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,60 @@
+
+## Features
+
+Let's start by looking at some of the coolest Mac-specific features of the app, sensors for all your webcams and microphones. Each camera and microphone has its own `binary_sensor` showing whether it is active or not. These can enable some really useful automations, especially for those home working at the moment. You could automatically turn off the radio when answering a call or close the blinds behind you to improve your video quality. To see just how useful this can be in the real world, check out this video of how our very own Frenck is using these sensors in his streaming set up.
+
+TODO: ADD VIDEO EMBED HERE
+
+One huge advantage of running on a Mac compared to a mobile device is the much larger battery. This means we are not constrained by battery saving measures and can address one of the most common gripes with the iOS app, update intervals. On a Mac entity updates are triggered immediately when something changes. You will see this reported by the `sensor.DEVICE_NAME_last_update_trigger` reporting `Signaled`.
+
+### Notifications
+
+Just like the iOS app, you can send notifications to your Mac with services like `notify.mobile_app_DEVICE_NAME`. One small difference is that [critical notifications](https://companion.home-assistant.io/docs/notifications/critical-notifications) are not yet available for the Mac app. However, all our other notifications features like actionable notifications work on the Mac app. To see what is possible, take a look at [the docs](https://companion.home-assistant.io/).
+
+### Interface
+
+Your Mac isn't a mobile so you shouldn't be forced to use an interface designed for a mobile. In the Mac app the App Configuration page has been removed from Home Assistant's sidebar and instead the configuration options and preferences are in the Home Assistant menu on the menu bar right where you'd expect to find them for any other app and all the standard shortcuts work too (just hit `cmd + ,` for preferences). Here you will also find an option to manually send an update to Home Assistant and you can even open extra Home Assistant windows! Finally, on the menu bar you can also find a new Actions menu where you can see all your [actions](https://companion.home-assistant.io/docs/core/actions) and fire them.
+
+### Big Sur Features
+
+Home Assistant Companion for Mac already supports widgets in Big Sur. Right now we have an Actions Widget where you can have up to eight actions. You can also create multiple widgets with different sets of actions. If you have an idea for other widgets you'd like to see, pop over to [the community forums and let us know](https://community.home-assistant.io/t/what-kind-of-ios-14-widgets-would-you-like-to-see/211112/14).
+
+<p class='img'>
+<img src='/images/blog/2020-09-18-mac-companion/actions-widget.png' alt='Screenshot of a large Home Assistant Actions widget in Big Sur'></a>
```

Comment:
Would it be possible to show this screenshot in its context? As in, is this in the sidebar? Can we maybe include a bit more screen.

Reply:
More like this? 
<img width="474" alt="Screenshot 2020-09-16 at 21 46 59_crop" src="https://user-images.githubusercontent.com/12411302/93394075-84b70f00-f86b-11ea-8735-699269da89e2.png">

Reply:
Yes, but without Home Assistant window next behind it maybe. Wonder if you can show some other controls too, so you see what the control pane looks like ?

Reply:
Sure, will add a couple of other widgets (there aren't so many available yet)

Reply:
![image](https://user-images.githubusercontent.com/12411302/93449695-f4a9b180-f8cc-11ea-826a-1d7e9a81f0d5.png)
Pretty much all the other widgets available are either from my pictures or calendar